build.xml

```diff
@@ -27,8 +27,9 @@
     </fail>
 
     <property environment="env"/>
-    <property file="build.properties" />
-    <property file="build.properties.default" />
+    <property file="build.properties"/>
+    <property file="build.properties.default"/>
```

Comment:
nit: a few lines in this file change for only whitespace. remove from patch please.

Reply:
I agree with Mick, the patch should only contain changes related to the problem. Formatting is definitely not on that list.